packages/lodestar-validator/src/services/block.ts

```diff
@@ -55,7 +55,8 @@ export default class BlockProposingService {
   public start = async (): Promise<void> => {
     //trigger getting duties for current epoch
     const slot = this.provider.getCurrentSlot();
-    this.onNewEpoch(computeEpochAtSlot(this.config, slot));
+    // ### Todo: Unknown promise duration
+    await this.onNewEpoch(computeEpochAtSlot(this.config, slot));
```

Comment:
What are the consequences of awaiting this function in the validator start sequence?

Reply:
I don't think they are any

Reply:
Well if attestation is on first slot, start method would wait quite a lot of time to resolve. Errors in those methods ain't suppose to propagate up. it should be handled and this service should just loop and wiat for next duty